test/engagement-workflow.e2e-spec.ts

```diff
@@ -1,212 +1,85 @@
+    await app.close();
+  });
+  beforeEach(async () => {
+    await projectManager.login();
   });
 
-  describe('should test engagement status Active when Project is made Active', () => {
-    it('translation', async function () {
-      // --- Translation project
-      const transProject = await createProject(app, {
-        type: ProjectType.MomentumTranslation,
-      });
-      const langEngagement = await createLanguageEngagement(app, {
-        projectId: transProject.id,
+  it('Start Late', async () => {
+    const lateEng = await runAsAdmin(app, async () => {
+      const lateEng = await createLanguageEngagement(app, {
+        projectId: project.id,
       });
-      await runAsAdmin(app, async () => {
-        const fundingAccount = await createFundingAccount(app);
-        const location = await createLocation(app, {
-          fundingAccountId: fundingAccount.id,
-        });
-        const fieldRegion = await createRegion(app);
-        await updateProject(app, {
-          id: transProject.id,
-          primaryLocationId: location.id,
-          fieldRegionId: fieldRegion.id,
-        });
-        for (const next of stepsFromEarlyConversationToBeforeActive) {
-          await changeProjectStep(app, transProject.id, next);
-        }
-        await changeProjectStep(app, transProject.id, ProjectStep.Active);
-      });
-      const lEngagementStatus = await getCurrentEngagementStatus(
-        app,
-        langEngagement.id,
-      );
-      expect(lEngagementStatus.status.value).toBe(EngagementStatus.Active);
+      return lateEng;
     });
-    it('internship', async function () {
-      // --- Internship project
-      const internProject = await createProject(app, {
-        type: ProjectType.Internship,
-      });
-      const internEngagement = await createInternshipEngagement(app, {
-        projectId: internProject.id,
-      });
-      await runAsAdmin(app, async () => {
-        const fundingAccount = await createFundingAccount(app);
-        const location = await createLocation(app, {
-          fundingAccountId: fundingAccount.id,
-        });
-        const fieldRegion = await createRegion(app);
+    const eng = await EngagementWorkflowTester.for(app, lateEng.id);
+    expect(eng.state).toBe(EngagementStatus.InDevelopment);
 
-        await updateProject(app, {
-          id: internProject.id,
-          primaryLocationId: location.id,
-          fieldRegionId: fieldRegion.id,
-        });
-        for (const next of stepsFromEarlyConversationToBeforeActive) {
-          await changeProjectStep(app, internProject.id, next);
-        }
-        await changeProjectStep(app, internProject.id, ProjectStep.Active);
-      });
-      const lEngagementStatus = await getCurrentEngagementStatus(
-        app,
-        internEngagement.id,
-      );
-      expect(lEngagementStatus.status.value).toBe(EngagementStatus.Active);
-    });
+    await controller.login();
+    await eng.executeByLabel('Approve');
```

Comment:
@CarsonF , I executeByLabel('Approve') flows down to the method `findTransition` in the workflow.tester.ts file where it fails to find a transition and throws the exception, `This transition is not available`.  I tried to debug this and the rest of the CI errors, but I'm not understanding why the transitions are not being found.